src/axom/sidre/spio/interface/c_fortran/typesSPIO.h

```diff
@@ -4,7 +4,8 @@
 // Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
 // other Axom Project Developers. See the top-level COPYRIGHT file for details.
 //
-// SPDX-License-Identifier: (BSD-3-Clause)
+// SPDX-License-Identifier (BSD-3-Clause)
```

Comment:
It looks like other uses of SPDX have the colon.

Reply:
This is what was generated running shroud.  I don't know if there's inconsistent behavior somewhere.

Reply:
The YAML file has the colon.  I'll check it out.  Don't let it hold up the pull request.